test/manual/kerberos.test.js

```diff
@@ -0,0 +1,352 @@
+'use strict';
```

Comment:
Can you generally spruce this file up a bit? 
- use `let`/`const` everywhere
- switch to using `chai` instead of the legacy test assert helpers
- `it.skip` the tests we don't think are useful (or better yet, delete them)
  - for instance, we no longer have a `reconnect` event for topologies
- nit: fix the test titles to be more consistent (lower case them, don't mention redundant info "should correctly", or "on MongoClient")
-